dashboard/settings.py

```diff
@@ -119,6 +121,9 @@ def path(*parts):
 # https://docs.djangoproject.com/en/1.9/topics/i18n/
 
 LANGUAGE_CODE = 'en-us'
+LOCALE_PATHS = [
+    path('locale',),
```

Comment:
Not disagreeing at all, just curious as I learn more about Python. Is it common to add a trailing comma when using functions with variable-length arguments? I've seen it before. What's that about?